src/pydata_sphinx_theme/assets/styles/components/_search.scss

```diff
@@ -93,6 +95,8 @@
 
     i.fa-solid.fa-magnifying-glass {
       padding-right: 0.5rem;
+      position: absolute;
```

Comment:
I don't think these two are needed, since they should be inherited from the base-level CSS rules, no?

Reply:
the 3 of them were actually a duplicate. I also removed the padding-right as it was only added because the icon was outside